doc/source/whatsnew/v0.23.0.txt

```diff
@@ -530,6 +530,8 @@ I/O
 - Bug in :func:`DataFrame.to_parquet` where an exception was raised if the write destination is S3 (:issue:`19134`)
 - :class:`Interval` now supported in :func:`DataFrame.to_excel` for all Excel file types (:issue:`19242`)
 - :class:`Timedelta` now supported in :func:`DataFrame.to_excel` for xls file type (:issue:`19242`, :issue:`9155`)
+- Bug in :class:`pandas.io.stata.StataReader` raising ``AttributeError`` when ``value_labels()`` called with very old files. Now returns an empty dict (:issue:`19417`)
```

Comment:
We don't have an entry for `StataReader` in the API docs, but we do have `StataReader.value_labels`. So could you reword this as

    - Bug in :meth:`pandas.io.stata.StataReader.value_labels` raising an ``AttributeError`` when called on very old files. Now returns an empty dict (:issue:`19417`)

Reply:
Changed